packages/xo-remote-parser/src/index.spec.js

```diff
@@ -45,7 +45,7 @@ const data = deepFreeze({
     },
   },
   S3: {
-    string: 's3://AKIAS:XSuBupZ0mJlu%2B@s3-us-west-2.amazonaws.com/test-bucket/dir',
+    string: 's3://AKIAS:XSuBupZ0mJlu%2B@s3-us-west-2.amazonaws.com/test-bucket/dir?acceptSelfSigned=false',
```

Comment:
Add a new test instead, we want to make sure that entries without this param are still supported.